tests/migration/Core/V6_8/Migration1756068709FixCustomerAddressNameFieldLengthsTest.php

```diff
@@ -0,0 +1,61 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_8;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Exception;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelLifecycleManager;
+use Shopware\Core\Migration\V6_8\Migration1756068709FixCustomerAddressNameFieldLengths;
+
+/**
+ * @internal
+ */
+#[CoversClass(Migration1756068709FixCustomerAddressNameFieldLengths::class)]
+class Migration1756068709FixCustomerAddressNameFieldLengthsTest extends TestCase
+{
+    private Connection $connection;
+
+    protected function setUp(): void
+    {
+        $this->connection = KernelLifecycleManager::getConnection();
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function testMigration(): void
+    {
+        $migration = new Migration1756068709FixCustomerAddressNameFieldLengths();
+        static::assertSame(1756068709, $migration->getCreationTimestamp());
+
+        $migration->update($this->connection);
```

Comment:
please execute the migration twice in the unit tests, that way we can make sure that it is safe to rerun the migration when something fails e.g mysql has gone away between the first and second query in your migraion